web-extension/test/admin-ns.spec.js

```diff
@@ -5,6 +5,8 @@ import { NetworkCollection } from '../backend/network.js'
 import ConcurrentStorage from '../lib/concurrent-storage.js'
```

Comment:
Feel like this file is getting quite big and hard to grok. Perhaps too much is happening in admin ns and we could split some stuff into chunks. Not a blocking comment just, like, my opinion man.